server/schedule/filter/filters.go

```diff
@@ -672,6 +722,32 @@ func (f *specialUseFilter) Target(opt opt.Options, store *core.StoreInfo) bool {
 	return !f.constraint.MatchStore(store)
 }
 
+type alwaysAllowedFilter struct {
```

Comment:
IMO, we can remove this filter for simplicity.

Reply:
updated.